package.json

```diff
@@ -9,7 +9,7 @@
     "node": "22.12.0"
   },
   "dependencies": {
-    "@sentry/node": "~7",
+    "@sentry/node": "~9",
```

Comment:
This is probably not so important but I'd like to mention that latest Sentry [docs](https://docs.sentry.io/platforms/javascript/guides/express/install/commonjs/) says that we should initialize sentry SDK before requiring any other module to ensure errors from those module are captured as well. I am not sure if this recommendation came in version 8 or 9 or it was always there.

Reply:
Great catch, thanks!  I'll review these recommendations.

Reply:
It looks like the change was introduced in Sentry v8:

> To migrate your Express app to Sentry Node SDK `8.x`, you need to make the following changes:
>
> 1. Create an external file called `instrument.js` and move your `Sentry.init` call to it.
>
> Due to the way that auto instrumentation works in `8.x` of the Sentry Node SDK, it is required that you initialize Sentry before you require or import any other package. Any package that is required/imported before Sentry is initialized may not be correctly auto-instrumented.

https://github.com/getsentry/sentry-docs/blob/09e515bbdf1fc4d2372c4cf5c764cf18620ebf22/platform-includes/migration/javascript-v8/important-changes/javascript.express.mdx#L45

Reply:
It looks like v7 docs are here: https://github.com/getsentry/sentry-docs/blob/37831f9bb64b217355ef52aeefd9ff6722e0466c/docs/platforms/javascript/guides/express/index.mdx

> Sentry should be initialized as early in your app as possible:

Reply:
I've updated the PR to initialise Sentry as early as possible, and added documentation of why this is happening.